roles/common/tasks/crd_test.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify logging credentials "{{ item }}"
+  ansible.builtin.shell:
+    cmd: |
+      oc get crd "{{ item }}"
+  changed_when: false
+  register: output
+
+- name: |
+   [Test] Fail if CRD is not found
```

Comment:
I don't think this will work as expected. I haven't tested it, but it looks suspicious to me.
The status of this will either be "failed" or "skipped", never "pass"